app/controllers/accounts_controller.rb

```diff
@@ -1,9 +1,7 @@
 class AccountsController < ApplicationController
-  before_action :authenticate_user!
```

Comment:
@peterclark this looks like something we shouldn't do.  We require authentication everywhere unless explicitly skipped for public pages/controllers.  Can we make travis/rspec work with this in place?